server/views/pages/partials/printNotification/sentenceLength.njk

```diff
@@ -0,0 +1,20 @@
+{% macro sentenceLength(terms, label) %}
+    {% for term in terms %}
```

Comment:
possibly untested

Reply:
Already covered under - GET /view/:nomsId/calculation-summary/:calculationRequestId/printNotificationSlip?fromPage=view should generate page - Assertions of length 

<img width="771" alt="Screenshot 2024-05-16 at 14 15 34" src="https://github.com/ministryofjustice/calculate-release-dates/assets/165897256/496dc1b1-a71e-4ef6-8fdc-0c2ebb54e990">